scripts/format-changed

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+
+    echo -n \
+"Usage: $(basename "$0")
+
+Format changed scala files. Compares against develop, so results may be unpredictable
+if your branch isn't rebased on develop / if your develop isn't up-to-date.
+"
+}
+
+function format_changed_scala_files() {
+    files=$(git diff --name-only develop..HEAD | grep '\.scala')
+    echo "${files}"
+    scalafmt "${files}"
```

Comment:
This errored out for me when it found multiple files. The string of the multiple files is being passed to `scalafmt`, and it interprets it as a single file, which of course it can't find. I think you need to use `xargs` here to convert the list of files into first-class arguments.

Reply:
I'm specifically trying to avoid using xargs for jvm startup/shutdown reasons. I wound up quoting it only because shellcheck got mad at me for not quoting it, and it worked correctly without `${files}`. I'll have to think for a minute to see if I can make shellcheck happy and avoid xargs.